docs/index.md

```diff
@@ -1,33 +1,52 @@
 # <provider> Delphix Provider
 
-!> **NOTE**
-The Terraform Provider for Delphix requires the Delphix Continuous Data Platform and Data Control Tower (DCT).
+The Terraform Provider for Delphix enables customers to natively manage data-as-code along with their infrastructure.
```

Comment:
replace customers with you and their with your
The Terraform Provider for Delphix enables you to natively manage data-as-code along with your infrastructure.